test/features/lb/public_lb.feature

```diff
@@ -189,14 +190,14 @@ Feature: Test different tfvars configurations for the lb module with a public ti
         Then terraform resource 'aws_lb' 'default' has changed attributes
             | attr                       | value             |
             #----------------------------|-------------------|
-            | enable_deletion_protection | true              |
-            | enable_http2               | false             |
-            | idle_timeout               | 100               |
+            | enable_deletion_protection | ${bool:True}              |
+            | enable_http2               | ${bool:False}             |
+            | idle_timeout               | ${int:100}               |
             | ip_address_type            | <ip_address_type> |
         
         Examples: Different ip address types
             | ip_address_type |
             #-----------------|
             | dualstack       |
             | ipv4            |
-            
\ No newline at end of file
+            
```

Comment:
Please kill the extra whitespace :-D

Reply:
Just to clarify: do you not want ANY whitespace at the end of the files? I saw that I had several lines which is overkill, but GitHub complains when there isn't at least 1 newline character at the end of the file. Are we ignoring that?

Reply:
One is fine.